### ISSUE TITLE
Center toast

### ISSUE THREAD

**Issue by chitgoks** [CodeSeven/toastr#595]
There's only top-center and bottom-center

Please provide a center option horizontally and vertically.

**Comment by sekmenhuseyin** [comment]
Yes, it would be great if we can show message at the center of the screen

**Comment by xanisu** [comment]
same opinion here, it's a pity not to have this option

**Comment by ghost** [comment]
Here is a task
please check it: https://github.com/CodeSeven/toastr/issues/233 
@chitgoks
